src/Microsoft.SqlTools.ServiceLayer/Connection/ConnectionService.cs

```diff
@@ -115,7 +116,8 @@ internal IConnectedBindingQueue ConnectionQueue
 
         static ConnectionService()
         {
-            SqlColumnEncryptionAzureKeyVaultProvider sqlColumnEncryptionAzureKeyVaultProvider = new SqlColumnEncryptionAzureKeyVaultProvider(AzureActiveDirectoryAuthenticationCallback);
+            var tokenCredential = new DefaultAzureCredential();
+            SqlColumnEncryptionAzureKeyVaultProvider sqlColumnEncryptionAzureKeyVaultProvider = new SqlColumnEncryptionAzureKeyVaultProvider(tokenCredential);

```

Comment:
How was the user authenticated before in callback? I could not find any bearer token being attached to the Security token request, can you clarify? Just want to make sure this is a safe replacement.